.github/workflows/tests.yml

```diff
@@ -186,6 +186,14 @@ jobs:
         id: brew-test-bot-formulae
         working-directory: ${{ env.BOTTLES_DIR }}
 
+      - name: Debug
```

Comment:
Please don't adjust CI.

Reply:
Of course, it's temporary - I'll remove it when I find out what happens.
I could not reproduce this issue locally after initial patch.